imod/util.py

```diff
@@ -57,18 +57,28 @@ def to_datetime(s):
     return time
 
 
-def _groupdict(stem: str, pattern: str) -> Dict:
+def _custom_pattern_to_regex_pattern(pattern: str):
+    """
+    Compile iMOD Python's simplified custom pattern to regex pattern:
+    _custom_pattern_to_regex_pattern({name}_c{species})
+    is the same as calling:
+    (?P<{name}>[\\w.-]+)_c(?P<{species}>[\\w.-]+)).compile()
```

Comment:
How are you supposed to call it?

It's not valid Python, and the regular expression isn't valid either?

I think you mean:

`pattern = re.compile("?P<name>[\\w.-]+)_c(?P<species>[\\w.-]+)")

(So without the curly braces.)

Reply:
Done